regression/goto-cc-goto-analyzer/instrument_preconditions_locations/s2n_hash_inlined.c

```diff
@@ -0,0 +1,44 @@
+struct evp_md_ctx_st
+{
+  const void *digest;
+};
+struct s2n_evp_digest
+{
+  struct evp_md_ctx_st *ctx;
```

Comment:
It feels like this might be simplified a bit more, can this be replaced with the `const void *` field directly?

Equally well; if you are done with reducing this, that is also fine.

Reply:
Done